mapview/__init__.py

```diff
@@ -8,6 +8,10 @@
 MapView is a Kivy widget that display maps.
 """
 
+from kivy.utils import platform
+from kivy.app import App
+import os.path
```

Comment:
sorry but it seems you left these unneeded imports :grimacing:

Reply:
Oops, my mistake for overlooking those